tests/configure.test.ts

```diff
@@ -135,34 +134,35 @@ describe('configure', () => {
 
     it('should generate tsconfig config', async () => {
       const tsconfigContents = await readJsonC(appFolderTS, 'tsconfig.json');
+
       expect(Object.keys(tsconfigContents).sort()).toEqual(['compilerOptions']);
     });
 
     it(`should generate \`.gitignore\``, async () => {
       const ignoreContents = await readIgnore(appFolderTS, '.gitignore');
-      expect(ignoreContents.length).toEqual(9);
-      expect(ignoreContents).toContain(`.eslintrc`);
-      expect(ignoreContents).toContain(`.eslintcache`);
-      expect(ignoreContents).toContain(`.prettierrc`);
-      expect(ignoreContents).toContain(`tsconfig.json`);
-      expect(ignoreContents).toContain(`.storybook/main.js`);
-      expect(ignoreContents).toContain(`${skuConfig.target}/`);
-      expect(ignoreContents).toContain(`${skuConfig.storybookTarget}/`);
-      expect(ignoreContents).toContain(`${bundleReportFolder}/`);
-      expect(ignoreContents).toContain(`${coverageFolder}/`);
+
+      expect(ignoreContents).toMatchInlineSnapshot(`
+        [
+          ".eslintcache",
+          ".eslintrc",
+          ".prettierrc",
+          ".storybook/main.js",
+          "coverage/",
+          "foo/bar/",
+          "report/",
+          "storybook/foobar/",
+          "tsconfig.json",
+        ]
+      `);
     });
 
-    ['.eslintignore', '.prettierignore'].forEach((ignore) =>
-      it(`should generate \`${ignore}\``, async () => {
+    it.each(['.eslintignore', '.prettierignore'])(
+      'should generate $ignore',
+      async (ignore) => {
         const ignoreContents = await readIgnore(appFolderTS, ignore);
-        expect(ignoreContents.length).toEqual(6);
-        expect(ignoreContents).toContain('*.less.d.ts');
-        expect(ignoreContents).toContain(`.storybook/main.js`);
-        expect(ignoreContents).toContain(`${skuConfig.target}/`);
-        expect(ignoreContents).toContain(`${skuConfig.storybookTarget}/`);
-        expect(ignoreContents).toContain(`${bundleReportFolder}/`);
-        expect(ignoreContents).toContain(`${coverageFolder}/`);
-      }),
+
+        expect(ignoreContents).toMatchSnapshot();
+      },
```

Comment:
Thoughts on bringing this inline like the above snapshot?

Reply:
Ordinarily I would, but you can't have inline snapshots inside an `it.each` unfortunately.